config/cluster.json

```diff
@@ -161,5 +161,10 @@
     "FRiP_Genrich": {
         "threads": "16",
         "gres": "lscratch:400"
-    }
+    },
+    "bam2bedgraph": {
+        "threads": "6",
+        "gres": "lscratch:400"
+    },
+    "SEACR": {}
```

Comment:
Just a reminder to update seacr resource allocation information once the pipeline completes